arcgis-ios-sdk-samples/Display information/Show legend/MILLegendTableViewController.swift

```diff
@@ -28,7 +28,7 @@ class MILLegendTableViewController: UITableViewController {
         self.populateLegends(self.operationalLayers as AnyObject as! [AGSLayerContent])
     }
     
-    func populateLegends(layers:[AGSLayerContent]) {
+    func populateLegends(_ layers:[AGSLayerContent]) {
```

Comment:
similar to other comments above about having argument name to clarify call site usage